### DISCUSSION TITLE
Does reqanroll supports gherkin?

### DISCUSSION THREAD

**Discussion by carmlw** [reqnroll/Reqnroll#258] [Q&A] [answered]
Hello! I have to migrate from Specflow to reqandroll, Can I use the same features? Could you suggest me the best way to do the process of migration?
Thank you

**Comment by ajeckmans** [comment, marked as answer]
Is there something not sufficiently over at https://docs.reqnroll.net/latest/guides/migrating-from-specflow.html ?
